helm/values/dev-values.yaml

```diff
@@ -0,0 +1,12 @@
+environment: dev
+branch: main #overridden for branch deploys
+
+service:
+  port: 3000
+
+replicaCount: 1
+
+image:
+  repo: quay.io/ukhomeofficedigital/
+  tag: latest # this value will be overridden by drone.yml
+  ecr_repo: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/callisto/
```

Comment:
Do we need this in each of the values files?

Reply:
For now I think we do, as we don't have a default value. I believe Stephen is going to look into Stephen centralising this, part of the helm chart work he is currently doing

Reply:
Is the ecr_repo referenced anywhere? I thought we are using quay now

Reply:
My understanding is that we're pushing to both. Did this change recently? If so I can remove the ecr_repo entry altogether

Reply:
I think you are right. I don't see any reference to ecr in drone build file. I only see quay. So I am going to remove the ecr_repo from the helm value files